src/ecdsa/keys.py

```diff
@@ -823,7 +823,7 @@ def from_pem(cls, string, hashfunc=sha1):
         """
         # the privkey pem may have multiple sections, commonly it also has
         # "EC PARAMETERS", we need just "EC PRIVATE KEY".
-        if PY3 and isinstance(string, str):
+        if not PY2 and isinstance(string, str):
```

Comment:
why `six.PY3` is unsafe?

Reply:
```python
if six.PY3:
    print("Python 3+ code")
else:
    print "Python 2 code" 
```

Where in six.py:

```python
PY3 = sys.version_info[0] == 3
```

When run on Python 4, this will run the Python 2 code!

Instead, use `six.PY2`.